src/js/gi/components/search/InstitutionFilterForm.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../../actions';
+
+import Checkbox from '../Checkbox';
+import RadioButtons from '../RadioButtons';
+import Dropdown from '../Dropdown';
+
+export class InstitutionFilterForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.renderTypeFilter = this.renderTypeFilter.bind(this);
+  }
+
+  renderTypeFilter() {
+    const type_facets = {
```

Comment:
I think this would fail the linter since it expects camelCase. Same for all the other variables in the other `render*` functions in this class.